src/content/reducers/types.js

```diff
@@ -29,8 +29,13 @@ export type ProfileViewState = {
   profile: Profile,
 };
 
+export type AppViewState = {
+  step: string,
```

Comment:
I'm not convinced that "step" is a good name for this, but I don't have a good alternative. "page"? "pageName"? "phase"? "stage"?